src/sagemaker_sklearn_container/serving.py

```diff
@@ -89,18 +91,32 @@ def _user_module_transformer(user_module):
                                    output_fn=output_fn)
 
 
+def import_module(module_name, module_dir):
```

Comment:
Food for thought. I think you guys can just remove all the logic for installing modules because SageMaker containers should handle it in the parent process. So remove this method and the code below.

https://github.com/aws/sagemaker-containers/blob/master/src/sagemaker_containers/_server.py#L79

Please test before trying this.

The reason is because the Gunicorn worker is executing this main function you guys defined below, which is where the installation is happening.